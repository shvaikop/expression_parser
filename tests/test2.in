5
3 + 4 - 2
2 + 4 * 5 / 3 + 17
7 * 7 / 2 % 2
12 % 5 + 15 * 6 % 7
1234 * 34567 + 6 / 3 * 5 - 5
