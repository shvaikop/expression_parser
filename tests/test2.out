5
25
0
8
42655683
